spec/tbd_tests_spec.rb

```diff
@@ -6255,93 +6255,125 @@
       expect(construction.layers.size).to eq(1)
       expect(construction.layers[0].handle.to_s).to eq(interior.handle.to_s)
 
-      # Fetch host surfaces.
-      ceiling = os_model.getSurfaceByName("Office Roof")
+      # Host spaces & surfaces.
+      sp1 = "Zone1 Office"
+      sp2 = "Zone2 Fine Storage"
+
+      z = "Zone2 Fine Storage ZN"
+
+      s1 = "Office Roof"          # Office surface hosting new TDD diffuser
+      s2 = "Office Roof Reversed" # FineStorage floor, above office
+      s3 = "Fine Storage Roof"    # FineStorage surface hosting new TDD dome
+
+      # Fetch host spaces & surfaces.
+      office = os_model.getSpaceByName(sp1)
+      expect(office.empty?).to be(false)
+      office = office.get
+
+      storage = os_model.getSpaceByName(sp2)
+      expect(storage.empty?).to be(false)
+      storage = storage.get
+
+      zone = storage.thermalZone
+      expect(zone.empty?).to be(false)
+      zone = zone.get
+      expect(zone.nameString).to eq(z)
+
+      ceiling = os_model.getSurfaceByName(s1)
       expect(ceiling.empty?).to be(false)
       ceiling = ceiling.get
+      sp = ceiling.space
+      expect(sp.empty?).to be(false)
+      sp = sp.get
+      expect(sp).to eq(office)
 
-      floor = os_model.getSurfaceByName("Office Roof Reversed")
+      floor = os_model.getSurfaceByName(s2)
       expect(floor.empty?).to be(false)
       floor = floor.get
-
-      roof = os_model.getSurfaceByName("Fine Storage Roof")
+      sp = floor.space
+      expect(sp.empty?).to be(false)
+      sp = sp.get
+      expect(sp).to eq(storage)
+
+      adj = ceiling.adjacentSurface
+      expect(adj.empty?).to be(false)
+      adj = adj.get
+      expect(adj).to eq(floor)
+
+      adj = floor.adjacentSurface
+      expect(adj.empty?).to be(false)
+      adj = adj.get
+      expect(adj).to eq(ceiling)
+
+      roof = os_model.getSurfaceByName(s3)
       expect(roof.empty?).to be(false)
       roof = roof.get
+      sp = roof.space
+      expect(sp.empty?).to be(false)
+      sp = sp.get
+      expect(sp).to eq(storage)
 
-      # A new, 1mx1m diffuser subsurface in Office Roof/Office Roof reversed.
```

Comment:
The fix here is to NOT generate a mirrored copy of the (subsurface) TDD diffuser in paired interzone surfaces. In all cases fixed, no errors or warnings reported in _eplusout.err_ !

Reply:
I see, yes you only need one diffuser for the TDD. I guess it's kind of weird but oh well.